deploy/example_catalog/cr-app-cdh5142cm.json

```diff
@@ -0,0 +1,268 @@
+{
+    "apiVersion": "kubedirector.bluedata.io/v1alpha1",
+    "kind": "KubeDirectorApp",
+    "metadata": {
+        "name" : "cdh5142cm"
+    },
+
+    "spec" : {
+        "systemdRequired" : true,
+        "persist_dirs" : ["/data"],
+        "capabilities" : [
```

Comment:
Do we know if this is the minimum required set?

Reply:
I think so. I took it from epic which is where i did of the experimenting for capabilities. Most of them are docker default capabilities. These are the additional ones that we are adding.

SYS_PACCT, SYS_RESOURCE, DAC_READ_SEARCH, NET_BROADCAST, SYS_PTRACE,IPC_LOCK,NET_RAW

IPC_LOCK may not be needed for CDH.

Reply:
Can/should we remove the ones that are already present?

I'm not sure actually... it may be good to be completely prescriptive here, since we don't know for sure what the underlying default caps will be.

Reply:
We can revisit this later to remove some caps that are not needed i think

Reply:
@joel looks good with reduced capabilities. We can revisit this to reduce further if needed